exercise/templates/exercise/_exercise_info.html

```diff
@@ -50,8 +50,19 @@ <h4 class="panel-title">{% trans "Exercise info" %}</h4>
             <dd class="exercise-info-required-points">{{ exercise.points_to_pass }}</dd>
             {% endif %}
 
+
+
             <dt>{% trans "Deadline" %}</dt>
             <dd class="exercise-info-deadline">{{ exercise.course_module.closing_time }}</dd>
+            {% if exercise|deviation_deadline:profile %}
+            <dt>{% trans "Extended deadline" %}</dt>
+            <dd class="exercise-info-deadline">
+              {{ exercise|deviation_deadline:profile }}
+              {% if exercise|has_deviation_penalty:profile|percent > 0 %}
```

Comment:
It looks like that `exercise|has_deviation_penalty:profile` returns a boolean, so how can you call the `percent` filter after it?